Marlin/macros.h

```diff
@@ -108,6 +108,8 @@
 #define HYPOT2(x,y) (sq(x)+sq(y))
 #define HYPOT(x,y) sqrt(HYPOT2(x,y))
 
+#define SIGN(a) ((a>0) - (a<0))
```

Comment:
This should be either: `((a > 0) - (a < 0))` or `((a>0)-(a<0))`, whichever is most preferred.